src/client/connection.ts

```diff
@@ -74,16 +71,16 @@ function getAgent(url: string, config: ConnectionOptions): Agent | undefined {
       (value) => value == null,
     )
     const proxyOptions = { ...parsedProxyURL, ...proxyOverrides }
-    let HttpsProxyAgent
+    let HttpsProxyAgent: new (opt: typeof proxyOptions) => Agent
     try {
-      // eslint-disable-next-line max-len -- Long eslint-disable-next-line TODO: figure out how to make this nicer
-      // eslint-disable-next-line import/max-dependencies, @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-require-imports, node/global-require, global-require, -- Necessary for the `require`
+      // eslint-disable-next-line max-len -- Long eslint-disable-next-line required.
```

Comment:
Similarly didn't know how to resolve, as theres not really a good way to do this in typescript.

Reply:
@ledhed2222 I believe did this by doing a normal disable and then re-enabling it below. idk which one is preferable